CONTRIBUTING.md

```diff
@@ -80,6 +114,32 @@ A couple of things you need to remember when editing already existing rules:
 - Try to add tests to cover the changes introduced, no matter if that's
   a bug fix or a new feature.
 
+## Writing Tests
+
+When writing tests for a new or existing rule, please make sure to follow these guidelines:
+
+###Write real-ish scenarios
+
+Since the plugin will report differently depending on Testing Library package is imported and Shared Settings are enabled, writing more realistic scenarios is pretty important. Ideally, you should:
```

Comment:
```suggestion
Since the plugin will report differently depending on which Testing Library package is imported and what Shared Settings are enabled, writing more realistic scenarios is pretty important. Ideally, you should:
```